ote/resources/public/language/en.edn

```diff
@@ -922,6 +922,16 @@ You can also draw the operating area or point to the map with drawing tools. You
   :published-types {:YES "Published" :NO "Not published" :ALL "All"}
   :interface-formats {:GTFS "GTFS" :Kalkati.net "Kalkati.net" :ALL "All"}}
 
+ :transit-changes
```

Comment:
Other sections have "-page" at the end

Reply:
True. Logic for dropping it was I suppose that strings are related to the "domain" and reused both on transit-changes page and transit-visualization page.